### PR TITLE
Adds ability to change owner of a proxy admin to set-admin command

### PR DESCRIPTION
**Pull request by ylv-io**
* Adds required functionality to a `set-admin` command in the following way:
`zos set-admin <new-proxy-admin-address>`
* Use interactive check to prevent admin address input errors.
* Adds tests to support required changes.

### CODE REVIEW

File: packages/lib/test/src/ProxyAdmin.test.js [hunk 1]
```diff
@@ -11,8 +11,8 @@ const ImplV1 = Contracts.getFromLocal('DummyImplementation');
 const ImplV2 = Contracts.getFromLocal('DummyImplementationV2');
 const ProxyAdminContract = Contracts.getFromLocal('ProxyAdmin');
 
-contract('ProxyAdmin class', function(accounts) {
-  const [_, proxyAdminOwner, newAdmin, otherAccount] = accounts.map(utils.toChecksumAddress);
+contract.only('ProxyAdmin class', function(accounts) {
```
Comment (spalladino) [inline packages/lib/test/src/ProxyAdmin.test.js]: Watch out! By the way, we need to add something that makes the CI fail if there is a `.only` in the specs. Or a git hook that runs on commit and checks it.

File: packages/cli/test/commands/setAdmin.test.js [hunk 2]
```diff
@@ -3,22 +3,26 @@ require('../setup')
 
 import { stubCommands, itShouldParse } from './share';
 
-describe('set-admin command', function() {
+describe.only('set-admin command', function() {
```
Comment (spalladino) [inline packages/cli/test/commands/setAdmin.test.js]: Remember to remove the `.only` from the tests!

File: packages/lib/test/src/ProxyAdmin.test.js (hunk 1 above)
Comment (ylv-io) [reply packages/lib/test/src/ProxyAdmin.test.js]: Would great to have cli to catch `.only`. Funny enough I was thinking the same.

File: packages/cli/src/commands/set-admin.ts [hunk 3]
```diff
@@ -7,29 +8,48 @@ import ConfigVariablesInitializer from '../models/initializer/ConfigVariablesIni
...
+    newAdmin = contractFullNameOrAddress;
+    contractFullNameOrAddress = '';
+  }
+
+  if (!options.force) {
+    const answers = await inquirer.prompt([
+      {
+        name: 'address',
+        type: 'string',
+        message: 'Please tripe check your address and type the last 4 letters of the new admin address. If you provide a wrong address you will lose control forever.',
```
Comment (spalladino) [inline packages/cli/src/commands/set-admin.ts]:
```suggestion
        message: 'Please double check your address and type the last 4 characters of the new admin address. If you provide a wrong address, you will lose control over your contracts.',
```

File: packages/cli/src/commands/set-admin.ts [hunk 4]
```diff
@@ -7,29 +8,48 @@ import ConfigVariablesInitializer from '../models/initializer/ConfigVariablesIni
...
+      }
+    ]);
+    if (answers.address !== newAdmin.slice(-4)) {
+      throw Error('Addresses do not much. Pay attention to your address. If you provide wrong address you will lose control forever.');
```
Comment (spalladino) [inline packages/cli/src/commands/set-admin.ts]:
```suggestion
      throw Error('Last 4 characters of the admin address do not match');
```
Keep an eye on spelling for `match`!

File: packages/cli/package-lock.json [hunk 5]
```diff
@@ -1,6639 +1,6099 @@
...
-		},
-		"dom-walk": {
-			"version": "0.1.1",
-			"resolved": "https://registry.npmjs.org/dom-walk/-/dom-walk-0.1.1.tgz",
-			"integrity": "sha1-ZyIm3HTI95mtNTB9+TaroRrNYBg="
-		},
-		"domelementtype": {
-			"version": "1.3.1",
-			"resolved": "https://registry.npmjs.org/domelementtype/-/domelementtype-1.3.1.tgz",
-
```
Comment (spalladino) [inline packages/cli/package-lock.json]: This is odd - why this change on the CLI package-lock? If there were no changes to dependencies, maybe remove it from the PR?

Comment (ylv-io) [reply packages/cli/package-lock.json]: Hm. Strange. Should be no changes in package-lock at all.
